qtdialogs/DlgSetupManager.py

```diff
@@ -0,0 +1,1303 @@
+         if isinstance(stateVal, int):
+            if stateVal == WALLET_STATE_LEGACY:
+               label = self.tr('Legacy')
+               isLegacy = True
+            elif stateVal == WALLET_STATE_ENCRYPTED:
+               label = self.tr('Encrypted')
+               isEncrypted = True
+            elif stateVal == WALLET_STATE_LOADED:
+               # treat loaded as ready for this setup view
+               label = self.tr('Ready')
+            elif stateVal in (WALLET_STATE_MIGRATED, 4):
+               label = self.tr('Ready')
+            elif stateVal == WALLET_STATE_UNKNOWN:
+               label = self.tr('Unknown')
+         else:
+            s = str(stateVal).lower()
+            if s == 'legacy':
+               label = self.tr('Legacy')
+               isLegacy = True
+            elif s == 'encrypted':
+               label = self.tr('Encrypted')
+               isEncrypted = True
+            elif s == 'loaded':
+               label = self.tr('Ready')
+            elif s in ('migrated', 'ready'):
+               label = self.tr('Ready')
+            elif s == 'unknown':
+               label = self.tr('Unknown')
+      except Exception:
+         # Default to Ready on any unexpected schema
+         label = self.tr('Ready')
+         isLegacy = False
+         isEncrypted = False
+
+      return (label, isLegacy, isEncrypted)
+
+   def migrateWallet(self, walletPath):
+      """Start migration using clean dependency injection pattern."""
+      try:
+         # Import wallet data through bridge
+         walletData = TheBridge.utils.importWallet(walletPath)
+
+         # Launch migration dialog with setup manager as main reference
+         # DlgSetupManager provides wallet management interface via self.wallets
+         dlg = DlgWalletMigration(self, self, walletPath, walletData)
+         result = dlg.exec_()
+
+         # Reload wallet list after migration dialog closes
+         TheSignalExecution.executeMethod(self.loadWalletList)
+
+         if result == QtWidgets.QDialog.Accepted:
+            LOGINFO("Wallet migration completed successfully")
+         else:
+            LOGINFO("Wallet migration cancelled by user")
+
+      except Exception as e:
+         QtWidgets.QMessageBox.warning(
+            self,
+            self.tr('Migration Failed'),
+            self.tr('Failed to start migration: {}').format(str(e))
+         )
+
+   def unlockWallet(self, walletId):
+      """Unlock wallet by staging and loading - bridge handles unlock
+      prompts."""
+      try:
+         # Stage the wallet - this triggers unlock prompts via bridge
+         # notifications
+         # Bridge will automatically handle passphrase prompts and unlock
+         # process
+         TheBridge.wltManager.stageWallet(walletId, True)
+
+         # Trigger wallet loading - bridge will prompt for unlock if needed
+         def loadWalletsAsync():
+            TheBridge.wltManager.loadWallets()
```

Comment:
This isn't correct: `loadWallets` loads wallet data. `listWallets` lists wallets available for loading.
THe purpose of SetupManager wallet tabs is to give the user an opportunity to curate the wallet list before loading. It naturally goes that SetupManager should not be calling `loadWallets`.